client/components/jetpack/daily-backup-status/status-card/backup-successful.jsx

```diff
@@ -73,7 +81,36 @@ const BackupSuccessful = ( { backup, deltas, selectedDate } ) => {
 				<div className="status-card__title">{ displayDateNoLatest }</div>
 			</div>
 			<div className="status-card__meta">{ meta }</div>
-			<ActionButtons rewindId={ actionableRewindId } />
+			{ isMultiSite && (
+				<div className="status-card__multisite-warning">
+					<div className="status-card__multisite-warning-title">
+						<Gridicon icon="notice-outline" />
+						{ translate( 'Your site, %(siteTitle)s, is a Multisite WordPress installation.', {
+							args: {
+								siteTitle: siteTitle,
+							},
+						} ) }
+					</div>
+					<p className="status-card__multisite-warning-info">
+						{ translate(
```

Comment:
<img width="980" alt="Screen Shot 2021-05-24 at 11 36 37 AM" src="https://user-images.githubusercontent.com/2810519/119392322-720b9a00-bc84-11eb-9784-e955b5004b2c.png">
This text looks like it could benefit from `preventWidows`. Additionally, the link seems a little short, maybe a couple more words could be added to it to make a bigger target?

Reply:
Thank you for reviewing @cbauerman! 🙂

> I am now getting the Backup Upsell ( in Calypso Blue, at least ) for a multi-site Jetpack site.

Great catch, thank you! I'll address that right away.

> This text looks like it could benefit from `preventWidows`. Additionally, the link seems a little short.

Yes, I agree. 👍

Reply:
Ok, fixed the Backup upsell showing, and changed the multisite message and link to match Erin Folletto's feedback below. 👍